src/Microsoft.DotNet.XHarness.Tests.Runners/AndroidApplicationEntryPoint.cs

```diff
@@ -34,30 +34,24 @@ public override async Task RunAsync()
             // we will write the normal console output using the LogWriter
             var logger = (Logger == null || options.EnableXml) ? new LogWriter(Device) : new LogWriter(Device, Logger);
             logger.MinimumLogLevel = MinimumLogLevel.Info;
-            var testAssemblies = GetTestAssemblies();
 
             var runner = await InternalRunAsync(logger);
+            ConfigureRunner(runner, options);
 
-
-            TestRunner.Jargon jargon = options.XmlVersion.ToTestRunnerJargon();
+            TextWriter writer = null;
```

Comment:
Not sure we are doing that here, but I am starting to get used to it 😅

```suggestion
            TextWriter? writer = null;
```